core-js-part5/task2.js

```diff
@@ -0,0 +1,6 @@
+function logMessage(message) {
+  console.log(message);
+  clearTimeout(timeoutId);
```

Comment:
clearTimeout should be called after initialising setTimeout (line 7) to demonstrate that the timeout callback function is prevented

Reply:
fixed!